### PR TITLE
server/cmd: add flag to print version information

### PR DESCRIPTION
**Pull request by Librazy**

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM
PTAL @huachaohuang

File: cmd/pd-server/main.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func main() {
 		log.Fatalf("initalize logger err %s\n", err)
 	}
 
-	server.PrintPDInfo()
+	server.LogPDInfo()
```
Comment (overvenus) [inline cmd/pd-server/main.go]: It's maybe too late to say that I personally prefer one line rather than two.

**Comment by overvenus** [comment]
Thank you @Librazy

**Pull request merged**